SpywareFilter/sections/general_url.txt

```diff
@@ -46,6 +46,7 @@
 ! Facebook
 /js/fb_tracking.js
 /fbpix-events-
+/fbevents.
```

Comment:
Actually, there is a script https://static.automexico.com/scripts/fbevents.min.js.gz?v=ea5a2775-d0f4-4580-9240-cda0ecaf6909.
So, you also should add 
```
/fbevents.min.js
```

Reply:
Another option is to create two rules
```
/fbevents.js
/fbevents.min.js
```